share/man/nitc.md

```diff
@@ -223,6 +223,11 @@ See the documentation of these specific modules for details.
 
     Currently, this only affect the android platform.
 
+`-g`, `--debug`
+:   Compile in debug mode.
+
+    Currently removes gcc optimizations.
```

Comment:
This also cause the nit_compile dir to not be removed so that C debugers can load the C source files.